packages/cli/test/commands/migrate/__snapshots__/e2e.ember-addon.test.ts.snap

```diff
@@ -0,0 +1,1399 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`migrate - ember addon: e2e > ember addon > should have the correct files in the addon components directory 1`] = `
+{
+  "data": [
```

Comment:
this is wrong. this should not be a Buffer

Reply:
fixed